common/changes/@microsoft/rush/octogonz-repro-for-4525_2024-02-21-03-40.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Fix a recent regression where `@rushstack/rush-sdk` did not declare its dependency on `@rushstack/terminal`",
```

Comment:
In fixing this test it felt like I was being overly paranoid, but once the test was working, it uncovered a related regression for `rush-sdk`. 😎